cfme/web_ui/paginator.py

```diff
@@ -15,6 +15,14 @@
 _page_cell = '//td//td[contains(., " of ")]'
 _check_all = Input("masterToggle")
 
+_records_re = r'\(\w+.*?(?P<offset>\d+).*?-.*?(?P<end>\d+).*?of.*?(?P<total>\d+)\)'
```

Comment:
The pagainator records regex was broken up into multiple parts and buried in functions, so I combined them. There's a little paranoia about what characters might exist where spaces should be; if we're confident that they will normally be spaces then this should also work:

`_records_re = r'\(\w+ (?P<offset>\d+) - (?P<end>\d+) of (?P<total>\d+)\)'`